docs/components/table.md

```diff
@@ -115,14 +125,30 @@ various parts of the table.
     <div v-else>Waiting</div>
 
     <KTable 
-      class="clickable-rows"
       :options="tableOptions"
+      is-clickable
       @row:click="actionRow" 
       @cell:mouseover="actionRow"
     />
   </div>
 </template>
-
+<template>
```

Comment:
this example is no longer showing

Reply:
Still not seeing this in the docs. Looks like its fixed on the other PR https://github.com/Kong/kongponents/pull/237/files#diff-4975fee17748787a06e99b3259d78e9fR311